src/MakePotCommand.php

```diff
@@ -704,20 +704,38 @@ protected function extract_strings() {
 			}
 
 			if ( ! $this->skip_theme_json ) {
-				// This will look for the top-level theme.json file, as well as
-				// any JSON file within the top-level styles/ directory.
-				ThemeJsonExtractor::fromDirectory(
+				JsonSchemaExtractor::fromDirectory(
 					$this->source,
 					$translations,
 					[
-						'schema'         => JsonSchemaExtractor::THEME_JSON_SOURCE,
-						'schemaFallback' => JsonSchemaExtractor::THEME_JSON_FALLBACK,
-						'include'        => $this->include,
-						'exclude'        => $this->exclude,
-						'extensions'     => [ 'json' ],
-						'addReferences'  => $this->location,
+						// Only look for theme.json files in any folder, nothing else.
```

Comment:
This means any file named `theme.json` at any level would be considered. This targets core's `wp-includes/theme.json`, Gutenberg's `theme.json`, and the top-level `theme.json`.

Reply:
Correct

Reply:
Hmm interesting edge case.

I don't see a need to change anything about the processing though. If `styles/theme.json` ends up being processed twice it will only be in the POT file once. There won't be any duplicates whatsoever.

Reply:
Nice, thanks for confirming 👍